### PR TITLE
feat: Expand on analysis logs description DOCS-696

### PR DESCRIPTION
**Pull request by nicklem**
Clarifies the function of the analysis logs tab

### :eyes: Live preview
* https://docs-696-document-the-view-logs-fea--docs-codacy.netlify.app/repositories/pull-requests/#viewing-analysis-logs
* https://docs-696-document-the-view-logs-fea--docs-codacy.netlify.app/repositories-coverage/pull-requests/#viewing-analysis-logs

### :construction: To do
- [x] SME review
- [x] Peer review
- [x] Screenshots

### CODE REVIEW

**Review by stefanvacareanu7 — APPROVED** [review]
LGTM!

File: docs/repositories/pull-requests.md [hunk 1]
```diff
@@ -47,6 +47,17 @@ Click a specific commit to see [detailed information about that commit](commits.
 
 ![Commits tab](images/pull-requests-tab-commits.png)
 
+## Viewing analysis logs
+
+Analysis logs can help you track and understand the performance of the tools and the timing of analyses on your pull requests.
+
+To access these logs, click the **View logs** button in the pull request status area. This opens a modal with two tabs:
```
Comment (claudiacarpinteiro) [inline docs/repositories/pull-requests.md:54]: We could add a link to the status area.
Also, replacing **View logs** "button" with "link", for consistency with the content in the [status section](https://github.com/codacy/docs/pull/2133/files#diff-2ddbad93640088a1b84e11fe6135383bd43e8f4b25e29ca5bca803a6c029f4f9R26).

```suggestion
To access these logs, click the **View logs** link in the [pull request status](#status) area. This opens a modal with two tabs:
```

File: docs/repositories/commits.md [hunk 2]
```diff
@@ -154,6 +154,14 @@ Depending on the languages being analyzed or if you haven't [set up coverage for
 <!-- vale on -->
 <!--tabs-end-->
 
+## Viewing analysis logs
+
+Analysis logs can help you identify the duration of each analysis phase and any potential issues.
+
+To access the analysis logs, click the **View logs** button in the commit status area. This opens a modal displaying execution times and outcomes for the tools used to analyze the commit.
```
Comment (claudiacarpinteiro) [inline docs/repositories/commits.md:161]: Same here:

```suggestion
To access the analysis logs, click the **View logs** link in the [commit status](#status) area. This opens a modal displaying execution times and outcomes for the tools used to analyze the commit.
```

File: docs/repositories-coverage/pull-requests.md [hunk 3]
```diff
@@ -57,6 +57,14 @@ Click a specific commit to see [detailed information about that commit](commits.
 
 ![Commits tab](images/pull-requests-tab-commits.png)
 
+## Viewing analysis logs
+
+Analysis logs can help you spot any issues with the coverage analysis, such as missing coverage reports.
+
+To access the coverage analysis logs, click the **View logs** button in the pull request status area. This opens a modal listing coverage reports received for the common ancestor commit and the head commit of the pull request.
```
Comment (claudiacarpinteiro) [inline docs/repositories-coverage/pull-requests.md:64]: And same here:

```suggestion
To access the coverage analysis logs, click the **View logs** link in the [pull request status](#status) area. This opens a modal listing coverage reports received for the common ancestor commit and the head commit of the pull request.
```

**Review by claudiacarpinteiro — APPROVED** [review]
@nicklem Looks very good! 👍 
I left some minor comments.

**Pull request merged**